subprojects/core/src/main/java/org/gradle/util/Clock.java

```diff
@@ -16,11 +16,13 @@
 
 package org.gradle.util;
 
+import org.gradle.internal.time.DefaultEventTimer;
+
 /**
- * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.Clock`.
+ * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.DefaultEventTimer`.
  */
 @Deprecated
-public class Clock extends org.gradle.internal.time.Clock {
+public class Clock extends DefaultEventTimer {
```

Comment:
This might be a breaking change for the build scan plugin. I believe it still relies on the code via `BuildRequestMetaData.getBuildTimeClock()`. Worth checking with them. We might want to introduce a replacement method they can use to switch over.